packages/core/src/provisioning.ts

```diff
@@ -351,4 +378,154 @@ export class BasicProvisioner implements Provisioner {
 
         return prov;
     }
+
+    getAccountIdFromScimRequest(httpReq: IncomingMessage) {
+        const url = new URL(`http://localhost${httpReq.url || ""}`);
+        const secretToken = url.searchParams.get("token") || "";
+
+        // TODO: find secret token in the DB
```

Comment:
I think it's ok to include the organisation id in the url, which will make it easier to find the `Org` and `OrgProvisioning` objects in the database. I.e.:

```sh
const orgId = url.searchParams.get("orgId");
const orgProvisioning = await this.storage.get(Org, orgId);
```